omnigibson/prims/entity_prim.py

```diff
@@ -1353,7 +1157,8 @@ def wake(self):
         Enable physics for this articulation
         """
         if self.articulated:
-            self._dc.wake_up_articulation(self._handle)
+            prim_id = PhysicsSchemaTools.sdfPathToInt(self.prim_path)
+            og.sim.psi.wake_up(og.sim.stage_id, prim_id)
```

Comment:
does this work? I don't think I've tested this functionality on 2023.1.0

Reply:
Testing looks OK. But I'm opening an issue to reconsider the whole sleep thing anyway.